.github/workflows/create-pr.yaml

```
@@ -0,0 +1,86 @@
+---
+name: create-pr
+
+on:
+  workflow_dispatch:
+  # TODO: schedule
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout infra-deployments
+      uses: actions/checkout@v3
+      with:
+        repository: redhat-appstudio/infra-deployments
+        ref: main
+        path: infra-deployments
+
+    - name: Checkout ec-cli
+      uses: actions/checkout@v3
+      with:
+        repository: hacbs-contract/ec-cli/
+        ref: batching
+        path: ec-cli
+
+    - name: Update ec-cli
+      run: ./hack/update-infra-deployments.sh ../infra-deployments
+      working-directory: ec-cli
+
+    - name: Checkout ec-policies
+      uses: actions/checkout@v3
+      with:
+        repository: hacbs-contract/ec-policies
+        ref: batching
+        path: ec-policies
+
+    - name: Update ec-policies
+      run: ./hack/update-infra-deployments.sh ../infra-deployments
+      working-directory: ec-policies
+
+    - name: Checkout enterprise-contract-controller
+      uses: actions/checkout@v3
+      with:
+        repository: hacbs-contract/enterprise-contract-controller
+        ref: batching
+        path: enterprise-contract-controller
+
+    - name: Update enterprise-contract-controller
+      run: ./hack/update-infra-deployments.sh ../infra-deployments
+      working-directory: enterprise-contract-controller
+
+    - name: Display diff
+      run: git diff
+      working-directory: infra-deployments
+
+    - name: Checkout infra-deployments-ci
+      uses: actions/checkout@v3
+      with:
+        path: infra-deployments-ci
+
+    - name: Create PR in infra-deployments
+      env:
+        EC_AUTOMATION_KEY: ${{ secrets.EC_AUTOMATION_KEY }}
+        DEPLOY_KEY: ${{ secrets.DEPLOY_KEY }}
+        # TODO: Is this right? Other repos have it as 32872589 but I can't figure out where
+        # that came from.
+        APP_INSTALL_ID: 29980719
```

```suggestion
        APP_INSTALL_ID: 32872589
```

This ends up being used to generate `GITHUB_TOKEN` needed for `gh pr create` against https://github.com/redhat-appstudio/infra-deployments/, the installation can be [seen here](https://github.com/apps/ec-automation/installations/32872589) (don't update any permissions there that'll trigger approvals in https://github.com/redhat-appstudio/)